RSDKUtils.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.homepage     = "https://github.com/rakutentech/ios-sdkutils"
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.source       = { :git => "https://github.com/rakutentech/ios-sdkutils.git", :tag => s.version.to_s }
-  s.platforms    = { :ios => '11.0', :watchos => '6.0' }
+  s.platforms    = { :ios => '12.0', :watchos => '6.0' }
```

Comment:
This is a breaking semver change so next version will need to be major.

Reply:
Do we need to create a ticket to release a new version of RSDKUtils? @donnie-jp

Reply:
yes @malayli 

you can ask PJM/PDM to support that.

Reply:
@donnie-jp We've already broken backward compatibility in previous commits